src/store/index.js

```diff
@@ -0,0 +1,9 @@
+import configureStoreProd from './configureStore.prod';
+import configureStoreDev from './configureStore.dev';
+
+export default function configureStore(state, env) {
```

Comment:
Hm, this file was a rename from configureStore.js; apparently it lost that.  (There wasn't much history in there).  Anyway, we're passing in the `env` here so that we don't have to use `process.env` or `configuration` directly.